project/metals.sbt

```diff
@@ -1,4 +1,4 @@
 // DO NOT EDIT! This file is auto-generated.
```

Comment:
This should be in the git ignore list.  Needs deleting, committing, ignoring and then final commit.